File: backend/Entities/Film.cs
namespace Cinema.Entities;

public class Film
{
    public string FilmId { get; set; }

    public string Title { get; set; }
    
    public string Category { get; set;} = string.Empty;

    public int Time { get; set; }

    public string Description { get; set; } = string.Empty;

    public string? Director { get; set; }
    
    public string? Actors { get; set; } = string.Empty;

    public string Country { get; set; } = string.Empty;

    public string? Poster { get; set; }

    public string? Trailer { get; set; }
}

File: frontend/src/types/Film.ts
export default interface Film {
    filmId: string,
    title: string,          // Tên phim
    category: string,       // Thể loại
    description: string,    // Mô tả
    director: string|null,  // Đạo diễn
    actors: string|null,    // Diễn viên, các nhau bởi dấu ',' ví dụ: "Nguyễn A,Nguyễn B,Nguyễn B"
    time: number            // Thời gian chiếu
    country: string,       // Ngôn ngữ
    poster: string|null,
    trailer: string|null
}
